src/modules/statistics/firmware-statistics/firmware-statistics.component.css

```diff
@@ -0,0 +1,3 @@
+.firmeware-list .selected, .tenant-list .selected{
+    background-color: #f5f5f5;
+}	
```

Comment:
Please pick a color form the c8y branding variables, instead of defining your own